changelog/261.removal.md

```diff
@@ -0,0 +1,2 @@
+Under each form of the domain, the definition of slot mappings without the keyword ``required_slots`` is now deprecated. Please always use the 
```

Comment:
I'd say this is an `improvement` as it makes the SDK compatible with the the new Rasa Open Source release. You're not deprecating anything here.